platform/build.gradle.kts

```diff
@@ -55,6 +55,9 @@ dependencies {
 }
 
 tasks.test {
+  // Enable parallel test execution
+  systemProperty("junit.jupiter.execution.parallel.enabled", true)
+//  systemProperty("junit.jupiter.execution.parallel.mode.default", "concurrent")
```

Comment:
Many of the tests in `platform` subproject are not thread safe.

Reply:
Should `junit.jupiter.execution.parallel.enabled` be `false` then?

Reply:
Yes. Set it to false.